### ISSUE TITLE
How can I fix @polkadot/api and its dependents version

### ISSUE THREAD

**Issue by aniiantt** [polkadot-js/api#1901]
I use yarn to install @polkadot/api and I specified version 1.2.1. package.json like this: 
`@polkadot/api: "1.2.1"`

Then I found that the @polkadot/api is 1.2.1, But its dependencies are all installed as 1.3.1. 

![image](https://user-images.githubusercontent.com/7029338/74892775-7137e880-53ef-11ea-8ab2-b2881d1e28fa.png)

Maybe it's because of `^`. I don't want the deps to be so unstable.  This can lead to many unknown errors.

**Comment by aniiantt** [comment]
Why can't the package.json of @polkadot/api be like this
```
"dependencies": {
    "@babel/runtime": "^7.8.4",
    "@polkadot/api-derive": "1.4.0-beta.10",
    "@polkadot/keyring": "2.4.1",
    "@polkadot/metadata": "1.4.0-beta.10",
    "@polkadot/rpc-core": "1.4.0-beta.10",
    "@polkadot/rpc-provider": "1.4.0-beta.10",
    "@polkadot/types": "1.4.0-beta.10",
    "@polkadot/util": "2.4.1",
    "@polkadot/util-crypto": "2.4.1",
    "bn.js": "^5.1.1",
    "eventemitter3": "^4.0.0",
    "rxjs": "^6.5.4"
  },
```
without `^`. 

This way the @polkadot/ API and its dependent versions will be exactly the same.

As changes occur, lerna will update the dependent version.

**Comment by jacogr** [comment]
Sure, can try fixed deps in the 1.4 release. Which may help management for some users.
